zrpc/internal/balancer/p2c/p2c.go

```diff
@@ -121,6 +121,28 @@ func (p *p2cPicker) buildDoneFunc(c *subConn) func(info balancer.DoneInfo) {
 		if td < 0 {
 			td = 0
 		}
+
+		// The function curve of y = x^(-x) is as follows.
```

Comment:
Replace this text curve with a chart link?

Reply:
Ok, under which repository of the zeromicro account should the image be placed?

Reply:
You can put it in zero-doc

Reply:
https://github.com/zeromicro/zero-doc/pull/179